index.html

```diff
@@ -28,6 +28,87 @@ <h1 class=" box-item headline"> Hey There.I'm Ahmed <br> <span id="headline-gree
         </ul>
     </section>
     </main>
+    <section class="works">
+        <div class="action-btn"><img src="img/Disabled.png" alt=""></div>
+        <h1 class="headline"> My Recent Works </h1>   
+        <div class="main-container">
+            <div class="grid-item"> 
+                <div class="flex-item1"><img src="img/Plaholder.png" alt="placeholder"></div>
```

Comment:
- [ ] Please note that the content of this section should use `flex` instead of `grid`👌 and I'd also suggest you use `article `for each project container instead of `div ` to make your page more semantic👏

Reply:
Done